recipes/cxxopts/all/conanfile.py

```diff
@@ -13,10 +14,35 @@ class CxxOptsConan(ConanFile):
     options = { "unicode": [True, False] }
     default_options = { "unicode": False }
     no_copy_source = True
-    _source_subfolder = "source_subfolder"
+    
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "14",
+            "gcc": "6",
```

Comment:
```suggestion
            "gcc": "5",
```

GCC 5 is more than enough for C++11

Reply:
https://github.com/conan-io/conan-center-index/pull/2905#issuecomment-692029125 thats what i tried originally :thinking:

Reply:
its was bad cmake in the test_package